runtime/src/iree/schemas/spirv_executable_def.fbs

```diff
@@ -10,6 +10,12 @@ namespace iree;
 file_identifier "SPVE";
 file_extension "spve";
 
+// Source code location denoted by a file name and line within that file.
+table SpirVFileLineLocDef {
+  filename:string;
+  line:int32;
```

Comment:
I also considered changing this flatbuffer to the `iree.spv` namespace instead of prefixing this table with `SpirV`

Reply:
yeah, iree.hal.spirv or something would be nice - we should get all the hal ones under iree.hal.* and can do that as a follow-up

Reply:
Will that break compatibility with existing flatbuffers, or are the namespaces just for the source code? Also yes, good for a follow-up and not something I'd want to fold into this PR itself.